src/GosuWrapper.cpp

```diff
@@ -193,6 +199,9 @@ extern "C" {
 
   const char* Gosu_default_font_name()
```

Comment:
Hmm, this is actually supposed to be a constant. I'll make a note that this should either be `Gosu::DEFAULT_FONT_NAME`, or that `Gosu::LICENSE` should actually be `Gosu::license()` — either way it should be consistent, and all of them can be constants in Ruby.